### ISSUE TITLE
convert README.md to asciidoc

### ISSUE THREAD

**Issue by rdmueller** [docToolchain/drawio-intellij-plugin#11]
I think it makes sense to have only one markup format within the project. 
The solution architecture (/src/docs) needs to be written in AsciiDoc, so let's also convert the README to AsciiDoc.

**Comment by ahus1** [comment]
Please note that the build.script reads a snippet from the AsciiDoc file, converts it to HTML and puts in in the plugins XML descriptor.

**Comment by rdmueller** [comment]
yes, just noticed it and worked on it. Then noticed that it's the same with the README.md :-D

**Comment by hediet** [comment]
I don't care, as long as it works on github and in the IntelliJ marketplace.

Afaik, vscode extensions are required to have markdown file for the readme, but this does not seem to be the case here.

What is the advantage of asciidoc over markdown? I never saw asciidoc before.

**Comment by rdmueller** [comment]
Markdown comes in many flavours while there is _one_ standard for AsciiDoc with a wide range of features, like tables, admonitions, includes etc. 
That is why I think that AsciiDoc is the better choice for technical docs
